back/db/migrate/20211806161359_add_code_to_pages.rb

```diff
@@ -0,0 +1,8 @@
+class AddCodeToPages < ActiveRecord::Migration[6.1]
+  def change
+    add_column :pages, :code, :string
+    ActiveRecord::Base.connection.execute "UPDATE pages SET code = 'custom'"
```

Comment:
I have a slight preference to not set it as default value (on the DB level) and instead force the application to provide a correct code (in order to avoid mistakes when accidentally forgetting to provide the code). The same reasoning applies for navbaritems, where there is also no default value for the code.